articles/client-platforms/_includes/_rules-introduction.md

```diff
@@ -1,6 +1,6 @@
 Rules are one of the cool features of Auth0. The reason behind that coolness is their flexibility, which gives you the ability to extend what Auth0 has to offer. They are just JavaScript functions which work like middlewares. To see a detailed description, please refer to [the full documentation](https://auth0.com/docs/rules).
 
-### Create a rule
+${title}
```

Comment:
Does not display correctly:

![image](https://cloud.githubusercontent.com/assets/11715799/16806909/22b7004e-491f-11e6-8a5e-7f7d2f383b2d.png)

https://auth0-docs-content-pr-1763.herokuapp.com/docs/quickstart/spa/angular2/06-rules